### ISSUE TITLE
FC Set resource/create is active for all templates

### ISSUE THREAD

**Issue by christianseel** [modxbot/migrate#7619]
christianseel created Redmine issue ID 7619

On my current MODX Revolution 2.2.1-pl install I have 2 FC sets. (One create and one update)

The create set seems to be active for all templates (even it's set to be only active for a specific template).
Maybe it's a FC issue or a general manager issue that a new unsaved resource has no real template set?

**Comment by splittingred** [comment]
splittingred submitted:

I can't replicate this - perhaps you could post what rules you have in place?

**Comment by christianseel** [comment]
christianseel submitted:

I've just replicated this in my "debug" installation. If you like I can give you access to the manager...

_It figures out that the issue exists just for TV based rules!_

To replicate it I created a new (second) template and created 2 new TVs that have access to both (default base template and the new second template). Then I created a new fc profile and one set for "resource/create" and set the template value to the second template.

I now added 2 rules for testing:
1st: set default value of the first TV to "foobar"
2nd: hide second TV

Both rules are active if I create a new resource (through the navigation site -> new document) with the default base template.
